rubrik_cdm/data_management.py

```diff
@@ -1334,16 +1334,16 @@ def _time_in_range(self, start, end, point_in_time):
         else:
             return start <= point_in_time or point_in_time <= end
 
-    def sql_live_mount(self, db_name, date, time, sql_instance=None, sql_host=None, mount_name=None, timeout=30):  # pylint: ignore
+    def sql_live_mount(self, db_name, sql_instance=None, sql_host=None, mount_name=None, date='latest', time='latest', timeout=30):  # pylint: ignore
         """Live Mount a database from a specified recovery point.
         Arguments:
             db_name {str} -- The name of the database to Live Mount.
-            date {str} -- The recovery_point date to recovery to formated as `Month-Day-Year` (ex: 1-15-2014).
-            time {str} -- The recovery_point time to recovery to formated as `Hour:Minute AM/PM` (ex: 1:30 AM).
         Keyword Arguments:
             sql_instance {str} -- The SQL instance name with the database you wish to Live Mount.
             sql_host {str} -- The SQL Host of the database/instance to Live Mount.
             mount_name {str} -- The name given to the Live Mounted database i.e. AdventureWorks_Clone.
```

Comment:
These three should be under Arguments since they do not have a default value

Reply:
Yep, makes sense